frontend/www/js/omegaup/components/course/ViewProgress.vue

```diff
@@ -1,39 +1,62 @@
 <template>
-  <div class="omegaup-course-viewprogress card">
-    <div class="card-header">
-      <h2>
-        <a v-bind:href="courseUrl">{{ course.name }}</a>
-      </h2>
-    </div>
-    <div class="card-body">
-      <table class="table table-striped">
-        <thead>
-          <tr>
-            <th>{{ T.wordsName }}</th>
-            <th class="score" v-for="assignment in assignments">
-              {{ assignment.name }}
-            </th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr v-for="student in students">
-            <td>
-              <a v-bind:href="studentProgressUrl(student)">{{
-                student.name || student.username
-              }}</a>
-            </td>
-            <td class="score" v-for="assignment in assignments">
-              {{ Math.round(score(student, assignment)) }}
-            </td>
-          </tr>
-        </tbody>
-      </table>
-    </div>
-    <!-- panel-body -->
-    <div class="card-footer">
-      {{ T.courseStudentsProgressExportToSpreadsheet }}:
-      <a v-bind:download="csvFilename" v-bind:href="csvDataUrl">.csv</a>
-      <a v-bind:download="odsFilename" v-bind:href="odsDataUrl">.ods</a>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-sm-10">
+        <div class="omegaup-course-viewprogress card">
+          <div class="card-header">
+            <h2>
+              <a v-bind:href="courseUrl">{{ course.name }}</a>
+            </h2>
+          </div>
+          <div class="card-body table-responsive">
+            <table class="table table-striped">
+              <thead>
+                <tr>
+                  <th>{{ T.wordsName }}</th>
+                  <th class="score" v-for="assignment in assignments">
+                    {{ assignment.name }}
+                  </th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr v-for="student in students">
+                  <td>
+                    <a v-bind:href="studentProgressUrl(student)">{{
+                      student.name || student.username
+                    }}</a>
+                  </td>
+                  <td class="score" v-for="assignment in assignments">
+                    {{ Math.round(score(student, assignment)) }}
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+      <div class="col-sm-2">
+        <div class="card sticky-top sticky-offset">
+          <div class="card-header p-1">
+            <p class="card-title text-sm-center mb-1">
+              {{ T.courseStudentsProgressExportToSpreadsheet }}
+            </p>
+          </div>
+          <div class="card-body">
+            <a
+              class="btn btn-primary btn-sm mb-1"
+              v-bind:download="csvFilename"
+              v-bind:href="csvDataUrl"
+              >{{ T.wordsExport }}.csv</a
```

Comment:
```suggestion
              >.csv</a
```

El nombre de archivo no es "Export", entonces es mejor que no parezca que así se llama. pero qué te parece en vez de "Export" poner un ícono?

Reply:
Lo del ícono, creo que hay para Bootstrap, el de csv https://mdbootstrap.com/docs/jquery/content/icons-list/ , pero ods , creo que habría que agregar la imagen.

O que opinas sobre solo cambiar el `T.wordsExport` a T.wordsExportTo`  así no habría que ponerlo como si fuera un nombre de archivo y la extensión.

Reply:
O por que no mejor cambiar el texto de la cajita a "Exportar tabla como:" y que los botones solo digan ".csv" y ".ods".

Reply:
Oh esa alternativa me agrada @heduenas  , si están de acuerdo la podemos implementar

Reply:
+1

Reply:
Que opinan?
![image](https://user-images.githubusercontent.com/12377916/86413477-ee6d4700-bc86-11ea-8788-e60c9ea2673c.png)

Reply:
Ahh, llegué tarde a la discusión. Aprovecho para comentarte que en cuestión de imágenes, estamos usando `vue-font-awesome`, ya hay ejemplos de como implementarlos en algunos componentes. Y los puedes consultar acá https://fontawesome.com/icons?d=gallery cuando requieras agregarlos.

Reply:
muchas gracias @pabo99 \o/